src/main/java/com/robinpowered/sdk/model/User.java

```diff
@@ -163,4 +163,16 @@ public Reference(int userId) {
             super(userId);
         }
     }
+
+    public static class DeviceId {
+        private Integer deviceId;
+
+        public DeviceId(Integer deviceId) {
+            this.deviceId = deviceId;
+        }
+
+        public Integer getDeviceId() {
+            return deviceId;
+        }
+    }
```

Comment:
Not sure why I felt compelled to make this an entire class. Changed to an `int` now.